getting-started/installation.md

```diff
@@ -14,18 +14,18 @@ installers can be found under:
 
 ## Manual Installation
 
+These instructions deploy a standalone version of Magic Mirror, for [Server Only](https://docs.magicmirror.builders/getting-started/installation.html#server-only) and [Client Only](https://docs.magicmirror.builders/getting-started/installation.html#client-only) installs please read the install specific instructions below
```

Comment:
please convert the urls here from absolute (https://docs.magicmirror.builds/<xyz> to relative (<xyz>)

Reply:
should be updated